File: math/three_consecutive_odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        ct = 0
        for n in arr:
            if n % 2 == 0:
                ct = 0
            else:
                ct += 1
            if ct == 3:
                return True
        return False

File: math/three_consecutive_odds.rb
# @param {Integer[]} arr
# @return {Boolean}
def three_consecutive_odds(arr)
    ct = 0
    arr.each do |n|
        ct += 1 if n.odd?
        ct = 0 if n.even?
        return true if ct == 3
    end
    false
end